lib/api/redfish-1.0/schemas.js

```diff
@@ -0,0 +1,56 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var injector = require('../../../index.js').injector;
+var redfish = injector.get('Http.Api.Services.Redfish');
+var controller = injector.get('Http.Services.Swagger').controller;
+var Promise = injector.get('Promise');    // jshint ignore:line
+var _ = injector.get('_'); // jshint ignore:line
+
+var listSchemas = controller(function(req, res) {
+    var options = redfish.makeOptions(req, res);
+    return Promise.props({
+        schemasObj: redfish.getSchemas()
+    }).then(function(data){
+        _.forEach(data.schemasObj, function(value, idx){
+            data.schemasObj[idx] = (value.split('/v1/'))[1];
+            //removing the .json extension
+            data.schemasObj[idx] = (data.schemasObj[idx].split('.jso'))[0];
```

Comment:
OK, just a reminder you didn't have any typo. 
Actually, your two splitting operation can be merged into a single one, I am OK with either way:
```javascript
> "/redfish/v1/myschema.json.1.0.0".split(/\/v1\/(\w+).json/)[1]
'myschema'
```